packages/@okta/vuepress-site/docs/guides/build-sso-integration/before-you-begin/openidconnect/prep.md

```diff
@@ -1,24 +1,22 @@
 ### Prepare an OIDC integration
 
-Okta uses a multi-tenant local credential system for OIDC. Each instance of your app integration inside a customer org has a separate set of OIDC client credentials that are used to access your application.
+Okta uses a [multi-tenant](/docs/concepts/multi-tenancy) local credential system for OIDC. Each instance of your app integration inside a customer org has a separate set of OIDC client credentials that are used to access your application.
 
-For example, consider a scenario where your app integration is added to 10 separate customer orgs. Seven of those customers create a single instance of your app integration. However, the other three customers each create two separate instances of your app integration so they can use different configuration options. This creates a total of 13 (7 + (3*2)) sets of client credentials for your application that you need to track.
+For example, consider a scenario where your app integration is added to 10 separate customer orgs. Seven of those customers create a single instance of your app integration. However, the other three customers each create two separate instances of your app integration so they can use different configuration options. This scenario creates a total of 13 sets of client credentials for your application that you need to track.
 
 This multi-tenant approach is different from other IdPs that use a global credential system, where a given application has the same customer credentials across all orgs.
 
-Before you create a new OIDC integration in Okta:
-
-<!-- [ian 2020.02.25] this step doesn't matter unless the user can choose something besides the web as a platform
-1. Decide which platform you'll be using for the integration:
-   * A web application is accessed through the browser and can remain running on a server that can store a secret safely.
-   * A native application resides on the end user's device.
-   * A single-page app (SPA) is a web application that is contained on a single web page. All code is retrieved when the page is loaded initially - the page doesn't reload or refresh. A SPA application cannot keep running on a server.
--->
+Now, before you create a new OIDC integration in Okta:
```

Comment:
```suggestion
Before you create a new OIDC integration in Okta:
```

Reply:
I added the "Now," here to differentiate between the multi-tenancy stuff at the top, but looking at it now, it's too informal and obviously doesn't flow the way I intended. Instead I've added h4 headers for the top "Multi-tenancy" and here "Prerequisites"